Formula/calyptia@3.rb

```diff
@@ -2,14 +2,14 @@
 # frozen_string_literal: true
 
 # This file was generated by GoReleaser. DO NOT EDIT.
-class Calyptia < Formula
+class CalyptiaAT3 < Formula
```

Comment:
Is this one good too "CalyptiaAT3"?

Reply:
yup, it was wrong before - it should match up with the filename and I checked the v1/v2 ones to confirm.